docs/source/reference/routines.rst

```diff
@@ -30,14 +30,15 @@ These functions cover a subset of
    ext
 
 
-CUB backend for reduction routines
-----------------------------------
+CUB/cuTENSOR backend for reduction routines
+-------------------------------------------
 Some CuPy reduction routines, including :func:`~cupy.sum`, :func:`~cupy.min`, :func:`~cupy.max`,
```

Comment:
Do we know why `min` and `max` are not rendered with hyperlinks in RTD?

Reply:
`numpy.{min,max}` are undocumented functions so cupy is not documenting them as well.

Reply:
Maybe we should point to amax/amin instead?

Reply:
What! I didn't know that...Interesting...Yeah sounds good.